lib/router.js

```diff
@@ -4056,6 +4056,9 @@ router.get('/fashionnetwork/headline/:country?', lazyloadRouteHandler('./routes/
 // router.get('/kbs/today/:language?', lazyloadRouteHandler('./routes/kbs/today'));
 // router.get('/kbs/news/:category?/:language?', lazyloadRouteHandler('./routes/kbs/news'));
 
+// 李佳芮的博客
+router.get('/blogs/ruijuzi', lazyloadRouteHandler('./routes/blogs/ruijuzi.js'));
+
 // Deprecated: DO NOT ADD ANY NEW ROUTES HERE
```

Comment:
This is really obvious: **DO NOT ADD ANY NEW ROUTES HERE**
Please follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)